src/Form/CompositeHelpRequestType.php

```diff
@@ -42,6 +44,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('confirm', CheckboxType::class, ['required' => true, 'mapped' => false, 'constraints' => [
                 new NotBlank(),
             ]])
+            ->add('captcha', HiddenType::class, [
```

Comment:
Would you call it something a bit less clear like `c` for instance, so that bots can't interpret it?